purr/styles/verdigris.json

```diff
@@ -0,0 +1,149 @@
+{
+	"$schema": "https://schemas.wp.org/wp/6.5/theme.json",
+	"version": 2,
+	"title": "Verdigris",
+	"settings": {
+		"color": {
+			"palette": [
+				{
+					"color": "#000000",
+					"name": "Base",
+					"slug": "base"
+				},
+				{
+					"color": "#32a852",
+					"name": "Base / 2",
+					"slug": "base-2"
+				},
+				{
+					"color": "#808080",
+					"name": "Contrast",
+					"slug": "contrast"
+				}
+			]
+		},
+		"typography": {
+			"fluid": true,
+			"fontFamilies": [
+				{
+					"fontFace": [
+						{
+							"fontFamily": "DM Sans",
+							"fontStretch": "normal",
+							"fontStyle": "normal",
+							"fontWeight": "700",
+							"src": [
+								"file:./assets/fonts/dm-sans/DMSans-Bold.woff2"
```

Comment:
I'm not sure it's a bug with your code but I don't see DM sans on the editor. I do see it applied in the frontend, though

Reply:
Its working for me in both editor as well as frontend too.

**1. Editor:** 

![editor](https://github.com/user-attachments/assets/72683350-5dcb-4ef3-bfc8-0203adfd6e65)

**2. Front-end:**

![frontend](https://github.com/user-attachments/assets/79a2e282-abe9-474f-8029-d282dce8c81c)